### DISCUSSION TITLE
Delete map style?

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#14177] [Maps Related]
I imported a custom map style (xml-file) but would now like to remove it again from the list of map styles in the app. Can I do this?

**Comment by vshcherb** [comment]
As of Today, you can do it only via File App Manager

**Reply by ERYpTION** [reply]
I have searched in my file manager but can't find the imported map styles? Is it a specific folder?

**Reply by vshcherb** [reply]
if you have an "osmand" folder access it should be in "rendering_styles". If you don't have on Android, you can try to change OsmAnd Storage to a different folder and check if you have access.

**Reply by vshcherb** [reply]
I've created an issue to follow up https://github.com/osmandapp/OsmAnd/issues/14178

**Reply by ERYpTION** [reply]
Changing storage was the solution!
